cmake_utils/cmake/project/cmake_options.cmake

```diff
@@ -33,6 +33,7 @@
 # - BUILD_LIBRARY               : ON
 #
 # - CODE_COVERAGE               : OFF
+# - ASAN_BUILD                  : OFF
```

Comment:
Please, add it also to README (we should check at some point that we have all avaialble options there)